pkg/service/instance.go

```diff
@@ -22,6 +24,7 @@ type Instance struct {
 	StatusReason                    string                 `json:"statusReason"`
 	Location                        string                 `json:"location"`
 	ResourceGroup                   string                 `json:"resourceGroup"`
+	ParentAlias                     string                 `json:"parentAlias"`
```

Comment:
Can we move this up above `Parent`... or else move `Parent` down below this? I just want to keep them together, so it's more obvious they're related.